data/jessachalla.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "Jessa Challa",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "I'm a developer. I want to make things that inspire curiosity and action.",
+  "avatar": "https://github.com/jessachalla.png",
+   "tags": [
+    "Technologist",
+    "Artist",
+    "Developer",
+    "JavaScript",
+    "Vue.js",
+    "Three.js",
+    "GIS",
+    "Always Learning"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/JessaChalla" },
+    { "icon": "FaGithub", "url": "https://github.com/jessachalla" }
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/jessachalla",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/JessaChalla",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Visit my website",
+      "url": "https://jessachalla.com",
+      "icon": "FaGlobe"
+    },
+    {
+      "name": "I'm happy to connect with you on LinkedIn",
+      "url": "https://www.linkedin.com/in/jessa-challa/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Buy me a coffee... Thank you!",
+      "url": "https://www.buymeacoffee.com/jessachalla",
+      "icon": "FaCoffee"
+    }
+  ],
```

```suggestion
  ]
```
Please remove this comma